src/edu/csus/ecs/pc2/clics/API202306/CLICSLanguage.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.LanguageUtilities;
+import edu.csus.ecs.pc2.core.model.Language;
+import edu.csus.ecs.pc2.core.util.IJSONTool;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Language
+ * Contains information about a language
+ *
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSLanguage {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private boolean entry_point_required;
+
+    @JsonProperty
+    private String entry_point_name;
+
+    @JsonProperty
+    private String [] extensions;
+
+    @JsonProperty
+    private CLICSCommand compiler;
+
+    @JsonProperty
+    private CLICSCommand runner;
+
+    /**
+     * Fills in the test case properties
+     *
+     * @param language The language being serialized
+     */
+    public CLICSLanguage(Language language) {
+        id = IJSONTool.getLanguageId(language);
+        name = language.getDisplayName();
+        entry_point_required = false;
```

Comment:
It's not clear to me why `entry_point_required` is being defaulted (assigned) to `false`.  Aren't there some languages that do require an entry point?   (Is the problem that we don't currently have any way to specify such a thing in PC2?)

Reply:
Correct.  And besides, `entry_point `is NOT required by any language we currently support.